src/pages/Settings.js

```diff
@@ -54,22 +54,12 @@ export default function Settings({ api, setUser }) {
     const levelKey = data.learned_language + "_cefr_level";
     const levelNumber = data[levelKey];
     setCEFR("" + levelNumber);
+    setUserDetails({
```

Comment:
should we remove the `api.modifyCEFRlevel` then? It does not seem to be used anywhere in the code anymore.

Reply:
sure. let's leave it for now then!